fw-child/resources/scss/_pages.scss

```diff
@@ -44,8 +53,9 @@
 }
 
 #hero-text-carousel {
-  margin-top: 5rem;
-  overflow: visible;
+  --hero-carousel-pb: 3rem;
```

Comment:
If the only use for this var is for this particular place, please use a sass var to avoid css var. This will lighten the output code. Only use css vars to override vars from vendor source.

Reply:
Changes applied. Thanks!